core/modules/widgets/action-deletetiddler.js

```diff
@@ -36,7 +36,7 @@ Compute the internal state of the widget
 */
 DeleteTiddlerWidget.prototype.execute = function() {
 	this.actionFilter = this.getAttribute("$filter");
-	this.actionTiddler = this.getAttribute("$tiddler");
+	this.actionTiddler = this.getAttribute("$tiddler",this.getVariable("currentTiddler"));
```

Comment:
I'm not sure that this default is sensible. We don't want to make it too easy to delete tiddlers. So I think it would make more sense to do nothing if the parameter is missing, and require the caller to be explicit about which tiddler they want to delete.